src/components/lyne-link-list/lyne-link-list.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  Component,
+  h,
+  Prop
+} from '@stencil/core';
+import { guid } from '../../global/guid';
+import { InterfaceLyneLinkListAttributes } from './lyne-link-list.custom.d';
+import { InterfaceTitleAttributes } from '../lyne-title/lyne-title.custom.d';
+
+/**
+ * @slot unnamed - Use this to document a slot.
+ */
+
+@Component({
+  shadow: true,
+  styleUrls: {
+    default: 'styles/lyne-link-list.default.scss',
+    shared: 'styles/lyne-link-list.shared.scss'
+  },
+  tag: 'lyne-link-list'
+})
+
+export class LyneLinkList {
+
+  private _guid: string;
+
+  /**
+   * The title text we want to show
+   * before the list
+   */
+  @Prop() public titleText?: string;
+
+  /**
+   * The semantic level of the title,
+   * e.g. 3 = h3
+   */
+  @Prop() public titleLevel?: InterfaceTitleAttributes['level'] = '2';
+
+  /**
+   * Choose the link list style. This
+   * does not refer to light or dark
+   * mode, but the background color
+   * on which the list is placed. Light
+   * and dark mode styling will be applied
+   * differently.
+   */
+  @Prop() public variant: InterfaceLyneLinkListAttributes['variant'] = 'positive';
+
+  public componentWillLoad(): void {
+    this._guid = guid();
+  }
+
+  public render(): JSX.Element {
+
+    let showTitle = true;
+    let additionalAttributes = {};
+
+    const id = `title-${this._guid}`;
+
+    if (this.titleText === '') {
+      showTitle = false;
+    } else {
+      additionalAttributes = {
+        'aria-labelledby': id
+      };
+    }
+
+    return (
+      // the role="list" is needed for voice over: https://bit.ly/3CDiZaG
```

Comment:
to be aria compliant, shouldn't we have `role="listitem"` on the chhildren as well? or is ios smart enough for that? (just doubting, because it seems to have issues with the ul anyway...)

Reply:
I thought the same but in the referenced article they were not used. I have tested it, with or without role="listitem" and the results are the same.